src/main/java/com/three_tech_solutions/slot_app/controllers/interfaces/PlanController.java

```diff
@@ -0,0 +1,13 @@
+package com.three_tech_solutions.slot_app.controllers.interfaces;
+
+import com.three_tech_solutions.slot_app.controllers.responses.PlanTypesResponse;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import java.util.List;
+
+@RequestMapping("/plan-types")
+public interface PlanController {
+    @GetMapping
+    List<PlanTypesResponse> getPlanTypes ();
```

Comment:
Los paréntesis deben ir pegados siempre al nombre del método

```suggestion
    List<PlanTypesResponse> getPlanTypes();
```